src/containers/Tenant/Diagnostics/Overview/TransferInfo/TransferInfo.tsx

```diff
@@ -0,0 +1,103 @@
+import {YDBDefinitionList} from '../../../../../components/YDBDefinitionList/YDBDefinitionList';
+import type {TEvDescribeSchemeResult} from '../../../../../types/api/schema';
+import {getEntityName} from '../../../utils';
+
+import {Credentials} from './Credentials';
+import i18n from './i18n';
+
+interface TransferProps {
+    data?: TEvDescribeSchemeResult;
+}
+
+/** Displays overview for Transfer EPathType */
+export function TransferInfo({data}: TransferProps) {
+    const entityName = getEntityName(data?.PathDescription);
+
+    if (!data) {
+        return (
+            <div className="error">
+                {i18n('noData')} {entityName}
+            </div>
+        );
+    }
+
+    const transferItems = prepareTransferItems(data);
+
+    return (
+        <Flex direction="column" gap="4">
+            <YDBDefinitionList title={entityName} items={transferItems} />
+        </Flex>
+    );
+}
+
+function prepareTransferItems(data: TEvDescribeSchemeResult) {
+    const transferDescription = data.PathDescription?.ReplicationDescription || {};
+    const state = transferDescription.State;
+    const srcConnectionParams = transferDescription.Config?.SrcConnectionParams || {};
+    const {Endpoint, Database} = srcConnectionParams;
+    const target = transferDescription.Config?.TransferSpecific?.Targets[0];
+    const srcPath = target?.SrcPath;
+    const dstPath = target?.DstPath;
+    const transformLambda = target?.TransformLambda;
+
+    const info: DefinitionListItem[] = [];
+
+    if (state) {
+        info.push({
+            name: i18n('state.label'),
+            content: <AsyncReplicationState state={state} />,
+        });
+    }
+
+    if (Endpoint) {
+        info.push({
+            name: i18n('srcConnection.endpoint.label'),
+            copyText: Endpoint,
+            content: <Text variant="code-inline-2">{Endpoint}</Text>,
+        });
+    }
+
+    if (Database) {
+        info.push({
+            name: i18n('srcConnection.database.label'),
+            copyText: Database,
+            content: <Text variant="code-inline-2">{Database}</Text>,
+        });
+    }
+
+    if (srcConnectionParams) {
+        info.push({
+            name: i18n('credentials.label'),
+            content: <Credentials connection={srcConnectionParams} />,
+        });
+    }
+
+    info.push({
+        name: i18n('srcPath.label'),
+        copyText: srcPath,
+        content: <Text variant="code-inline-2">{srcPath}</Text>,
+    });
+
+    info.push({
+        name: i18n('dstPath.label'),
+        copyText: dstPath,
+        content: <Text variant="code-inline-2">{dstPath}</Text>,
+    });
+
+    // TODO use true pre
+    info.push({
+        name: i18n('transformLambda.label'),
+        copyText: transformLambda,
+        content: (
+            <Text variant="code-2" whiteSpace="nowrap">
+                <pre>{transformLambda}</pre>
+            </Text>
+        ),
+    });
```

Comment:
For such queries it's better to use YQL Highlighter

Example: https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/containers/Tenant/Info/View/View.tsx#L12

<img width="901" alt="Screenshot 2025-03-05 at 18 36 59" src="https://github.com/user-attachments/assets/4bf53f7c-996c-4915-baa7-188b9162705d" />